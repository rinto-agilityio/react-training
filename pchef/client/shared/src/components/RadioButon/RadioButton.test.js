// Components
import RadioButton from '.'

describe('Button', () => {
  let props = {
    value: 'first',
    status: false,
  }
  it('Renders correctly', () => {
    const radioBtn = renderer.create(<RadioButton {...props} />).toJSON()

    expect(radioBtn).toMatchSnapshot()
  })

  it('Renders correctly radio button component with label', () => {
    props = {
      ...props,
      label: 'First',
    }
    const radioBtn = renderer.create(<RadioButton {...props} />).toJSON()

    expect(radioBtn).toMatchSnapshot()
  })

  it('Renders correctly radio button component with status true', () => {
    props = {
      ...props,
      status: true,
    }
    const radioBtn = renderer.create(<RadioButton {...props} />).toJSON()

    expect(radioBtn).toMatchSnapshot()
  })
})
