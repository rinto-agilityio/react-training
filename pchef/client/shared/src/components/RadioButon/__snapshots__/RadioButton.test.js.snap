// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Button Renders correctly 1`] = `null`;

exports[`Button Renders correctly radio button component with label 1`] = `null`;

exports[`Button Renders correctly radio button component with status true 1`] = `null`;
